src/app/login/login.component.html

```diff
@@ -0,0 +1,41 @@
+<div *ngIf="!name; else forminfo">
+  <form [formGroup]="rForm" (ngSubmit)="addPost(rForm.value)">
+    <div class="form-container">
+      <div class="row columns">
+        <h1>Login From</h1>
+        <div>
+        	<div>
+        		<label>Name
+          			<input type="text" formControlName="name">
+        		</label>
+        	</div>
+			<div class="alert" *ngIf="!rForm.controls['name'].valid && rForm.controls['name'].touched">{{ titleAlert }}</div>
```

Comment:
change the variable name from `titleAlert` to `nameAlert` .

Also maybe `username` makes more sense as a variable name here. Instead of just `name`

Reply:
changed